src/modules/72-templates-library/components/TemplateActivityLog/InfiniteScroll.tsx

```diff
@@ -144,7 +153,14 @@ export const useInfiniteScroll = (props: InfiniteScrollProps): InfiniteScrollRet
 
     loadItems()
     initialPageLoaded.current = true
-  }, [loadItems])
+  }, [])
+
+  useEffect(() => {
+    if (initialPageLoaded.current) {
+      offsetToFetch.current = 0
+      loadItems()
+    }
+  }, [searchTerm])
```

Comment:
Codacy has a fix for the issue: [React Hook useEffect has a missing dependency: 'loadItems'. Either include it or remove the dependency array.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232281)
```suggestion
  }, [loadItems, searchTerm])
```